### ISSUE TITLE
Forward and previous channel button does not change application title

### ISSUE THREAD

**Issue by Dartui** [mattermost/desktop#1563]
I confirm (by marking "x" in the [ ] below: [x]):

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**

![obraz](https://user-images.githubusercontent.com/2657856/115757948-e52ca280-a39e-11eb-9b38-0a7bfb946d13.png)

After clicking forward and previous channel buttons, application title does not change and stays the same.

**Environment**
- Operating System: Xubuntu 18.04 / Windows 10
- Mattermost Desktop App version: 4.6.2 (Xubuntu) / 4.6.1 (Windows)
- Mattermost Server version: 5.34.2

**Steps to reproduce**

1. Go on Channel 1 - application title changes to Channel 1
2. Go on Channel 2 - application title changes to Channel 2
3. Use Previous Channel button - channel is changed to Channel 1, but title says Channel 2

**Expected behavior**

Title changing with channel change

**Observed behavior**

**Possible fixes**

**Comment by amyblais** [comment]
@Dartui Are you able to reproduce the same on v4.7-rc2 https://github.com/mattermost/desktop/releases?

**Comment by Dartui** [comment]
@amyblais on Xubuntu, Mattermost 4.7-rc2 does not show channel name in application title at all. There is only "Mattermost Desktop App" in title. Later I will try it on Windows installation.

![obraz](https://user-images.githubusercontent.com/2657856/115884760-8d9a3f80-a44f-11eb-90b8-cab34f1a2f9b.png)

**Comment by Dartui** [comment]
Windows installation has the same application title, no matter of which channel is selected - Mattermost Desktop App.

To be honest I kinda liked seeing what channel I had open, but displaying same title all the time is fixing this issue.

**Comment by datn** [comment]
Linux application also always displays "Mattermost Desktop App", which is way too long for my task manager and ends up hiding notification numbers. Would appreciate a configurable dynamic title --

**Comment by amyblais** [comment]
cc @Willyfrog

**Comment by Willyfrog** [comment]
we removed using the title from the webapp in 4.7, in 5.0 it will be replaced by something more sensible and useful based on the state of the app.

**Comment by datn** [comment]
> we removed using the title from the webapp in 4.7, in 5.0 it will be replaced by something more sensible and useful based on the state of the app.

Good to know, thanks. 

> Can you provide a screenshot? from your description looks more like a WM issue rather than an app issue, but I would like to confirm.

I can't currently, because for my sanity, I actually used a binary editor to modify the app.asar fle to change <title>Mattermost Desktop App</title> to <title>Mattermost</title>. (Then I added an HTML comment to soak up the spaces) 

I hope this suffices:

```
#old 
0031F470  3A 22 3E 0A 20 20 20 20 3C 74 69 74 6C 65 3E 4D 61 74 74 65 72 6D 6F 73 :">.    <title>Mattermos
0031F488  74 20 44 65 73 6B 74 6F 70 20 41 70 70 3C 2F 74 69 74 6C 65 3E 0A 20 20 t Desktop App</title>.

#new
0031F470  3A 22 3E 0A 20 20 20 20 3C 74 69 74 6C 65 3E 6D 61 74 74 65 72 6D 6F 73 :">.    <title>mattermos
0031F488  74 3C 2F 74 69 74 6C 65 3E 20 20 3C 21 2D 2D 20 20 20 20 20 2D 2D 3E 20 t</title>  <!--     -->
```

**Comment by datn** [comment]
Thanks for this. Ideally a customizable title, but mainly this: a title should be as short as possible while distinguishing itself from others. I already know it's an app, and I don't need the desktop app distinguished from the mobile app because they run on separate platforms. So "Mattermost" is all that's needed.

**Comment by devinbinnie** [comment]
cc @esethna

**Comment by esethna** [comment]
@datn makes sense. Are you open to submitting a PR to shorten the title to "Mattermost" instead of "Mattermost Desktop App"
